pkgs/development/python-modules/concord232/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  requests,
+  stevedore,
+  prettytable,
+  pyserial,
+  flask,
+}:
+
+buildPythonPackage rec {
+  pname = "concord232";
+  version = "0.15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
```

Comment:
Shouldn't we use `fetchPypi`?
https://github.com/JasonCarter80/concord232/issues/22